content/en/docs/Installation/_index.md

```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
 helm install kyverno-policies kyverno/kyverno-policies -n kyverno
 ```
 
+### Notes for ArgoCD users
+
+When deploying this chart with ArgoCD you will need to enable `Replace` in the `syncOptions`, and you probably want to ignore diff in aggregated cluster roles.
+
+You can do so by following instructions in these pages of ArgoCD documentation:
+- [Enable Replace in the syncOptions](https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#replace-resource-instead-of-applying-changes)
+- [Ignore diff in aggregated cluster roles](https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#ignoring-rbac-changes-made-by-aggregateroles)
+
+ArgoCD uses helm only for templating but applies the results with `kubectl`.
+
+Unfortunately `kubectl` adds metadata that will cross the limit allowed by Kuberrnetes. Using `Replace` overcomes this limitation.
+
+Another option is to use server side apply, this will be supported in ArgoCD v2.5.
+
+Below is an example of ArgoCD application manifest that should work with this chart:
+```
```

Comment:
Always add a new line surrounding code blocks. Language should also be specified, here it's `yaml`.

Reply:
"Below is an example of an ArgoCD Application manifest that should work with the Kyverno Helm chart:"